docs/samples/product_recommendation_demo.ipynb

```diff
@@ -4,38 +4,40 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "# Demo Notebook: Feathr Feature Store on Azure\n",
+        "# Product Recommendation with Feathr\n",
```

Comment:
`on Azure` is removed from title, but it's still useful to readers of an open source project to know this sample requires Azure subscription.

Reply:
Added Azure back